packages/block-library/src/gallery/gallery.js

```diff
@@ -51,45 +42,13 @@ export const Gallery = ( props ) => {
 				'is-cropped': imageCrop,
 			} ) }
 		>
-			<ul className="blocks-gallery-grid">
-				{ images.map( ( img, index ) => {
-					const ariaLabel = sprintf(
-						/* translators: 1: the order number of the image. 2: the total number of images. */
-						__( 'image %1$d of %2$d in gallery' ),
-						index + 1,
-						images.length
-					);
+			<InnerBlocks
+				allowedBlocks={ [ 'core/image' ] }
+				__experimentalTagName="ul"
+				__experimentalAppenderTagName="li"
```

Comment:
need to work out how this works to see if we can use this instead of using the isList context to tell the child images to wrap in a <li>